### PR TITLE
Add betamax_parametrized_recorder and _session

### PR DESCRIPTION
**Pull request by hroncok**
Related to #112

No warnings yet.

### CODE REVIEW

File: betamax/fixtures/pytest.py [hunk 1]
```diff
@@ -49,6 +52,26 @@ def betamax_recorder(request):
 
 
 @pytest.fixture
+def betamax_recorder(request):
+    """Generate a recorder with a session that has Betamax already installed.
```
Comment (sigmavirus24) [inline betamax/fixtures/pytest.py]: The docstring should also mention that it does not include any pytest parameters for the test function in the name and that that will change by default in 1.0. Also that if they need that behaviour to check out `betamax_parametrized_recorder`.

**Comment by hroncok** [comment]
Amended. The failed test in Travis on 2.7 is unrelated (API rate limit exceeded).

**Comment by hroncok** [comment]
I've added a warning that is only happening when the user should care (i.e. when the cassette name would differ). No warning in the parametrized fixtures, because the user would get warnings in both, which I think is not nice.

My idea is:

 1. Merge this
 2. When 1.0.0 happens, change the default behavior (i.e. the betamax_recorder/session and betamax_parametrized_recorder/session will do the same thing), add DeprecationWarning for betamax_parametrized_recorder/session
 3. In reasonable future after 1.0.0, remove betamax_parametrized_recorder/session

**Comment by sigmavirus24** [comment]
So it's possible (and a good idea) to set filters for warning so they appear at most once. I'd like to do that for both now because I'd like to completely remove `betamax_parametrized_recorder` in 1.0 instead of keeping it around and *eventually* removing it. Does that sound fairer to you? (Sorry I didn't bring up the idea of filtering the warnings to only show up once.)

**Comment by hroncok** [comment]
Added the second warning. The default warnings filter should be fine.

**Comment by sigmavirus24** [comment]
@hroncok thanks so very much! This is fantastic!

**Comment by hroncok** [comment]
Thanks for this great tool.

**Pull request merged**